examples/svelte-apollo/src/AddAuthor.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import { mutation } from 'svelte-apollo'
+  import { gql } from '@apollo/client'
+  import { AUTHOR_LIST } from './graphql'
+
+  const ADD_AUTHOR = gql`
+    mutation ($name: String!) {
```

Comment:
Similar to the other mutation. Consider to use generic `$author: authors_insert_input!`

Reply:
Made changes.